### DISCUSSION TITLE
Add means to silence future warnings without opt-in

### DISCUSSION THREAD

**Discussion by uncaught** [remix-run/react-router#12321] [Proposals]
This is a follow up to #12245:

We cannot use a new feature, yet, due to dependencies.

Please give us the means to disable those warnings.

I'm fine with doing so individually, in a manner of "Yes, I've seen this feature flag now, but I cannot use it, yet. Thank you, be quiet now."

Maybe by explicitly setting the future flag to `false` rather than `true`, the warning could be skipped.

**Comment by mparsakia** [comment]
This is even showing up in our Jest tests.

**Comment by macko911** [comment]
Hey, this seems to have been addressed in https://github.com/remix-run/react-router/pull/12441 (yet to be released AFAICS)
